<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInstallFileLabelDialog</class>
 <widget class="QDialog" name="AddInstallFileLabelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Label Install File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_path">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="le_path">
     <property name="toolTip">
      <string>Ex: /usr/bin/*, /usr/share/__package_name__/data/image.png</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_file_type">
     <property name="text">
      <string>File Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="combo_file_type">
     <item>
      <property name="text">
       <string notr="true">EXECUTABLE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">LIBRARY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">DATA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">CONFIG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">DOC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">MAN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">INFO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">LOCALEDATA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">HEADER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ALL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="check_permanent">
     <property name="toolTip">
      <string>If permanent, don't delete during unisntall</string>
     </property>
     <property name="text">
      <string>Permanent</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_path</tabstop>
  <tabstop>combo_file_type</tabstop>
  <tabstop>check_permanent</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddInstallFileLabelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
